scripts/pypi_cleanup.py

```diff
@@ -13,9 +14,9 @@
 from requests.exceptions import RequestException
 
 # deletes old dev wheels from pypi. evil hack.
-# how many days to retain dev releases - all dev releases older than 10 days are deleted
-retain_days = 10
```

Comment:
This variable still appears to be used in the script so this doesn't work right now